src/ontology/edits/oeo-social.omn

```diff
@@ -651,7 +651,7 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/745",
     
     SubClassOf: 
         OEO_00000051,
-        <http://purl.obolibrary.org/obo/RO_0000056> some OEO_00040002
+        <http://purl.obolibrary.org/obo/RO_0000056> some OEO_00010082
```

Comment:
I understand that you changed this, because `market exchange` is a continuant, but `participates in` now demands a `occurant`/`process`.

The definition mentions the relation to the `market exchange`, so we should find either an (additional) axiom for that or change the definition. My first idea was `market participant member of some market exchange`, but that does not work as `member of` has range `person` but `market participant` is a `role`.

If we do not find a quick solution for this, I suggest that we accept the change of the axiom now and open an issue for that.

Reply:
So the market exchange is a continuant not a process, thus it cannot itself have participants. I added the axiom more indirectly by requiring the market exchange and the market participants both participate in some process, what do you think @l-emele ?

Reply:
`trade` is good too -- I changed it.